.github/workflows/matrix-tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: Matrix tests for interfaces
+
+on: workflow_dispatch
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    name: Excute Matrix tests
```

Comment:
A more descriptive name here would be nice: what about
'run the interface tests on all registered charms'